### PR TITLE
:recycle: Account basic fixes

### PR DESCRIPTION
**Pull request by zekiblue**

### CODE REVIEW

File: src/accounts/AccountV1.sol [hunk 1]
```diff
@@ -540,14 +540,14 @@ contract AccountV1 is AccountStorageV1, IAccount {
...
-    function accountManagementAction(address actionHandler, bytes calldata actionData, bytes calldata signature)
+    function flashAction(address actionTarget, bytes calldata actionData, bytes calldata signature)
         external
         nonReentrant
         onlyAssetManager
         notDuringAuction
         returns (address, uint256)
     {
-        if (!IRegistry(registry).isActionAllowed(actionHandler)) revert AccountErrors.ActionNotAllowed();
+        if (!IRegistry(registry).isActionAllowed(actionTarget)) revert AccountErrors.ActionNotAllowed();
```
Comment (j-vp) [inline src/accounts/AccountV1.sol:550]: can be removed alltogether (also in Registry) as part of the Account flow findings

**Comment by Thomas-Smets** [comment]
Will merge this in https://github.com/arcadia-finance/accounts-v2/pull/97, since it are both account fixes

File: src/accounts/AccountV1.sol [hunk 2]
```diff
@@ -381,6 +381,17 @@ contract AccountV1 is AccountStorageV1, IAccount {
...
     }
 
+    /**
+     * @notice Checks if the Account is healthy and still has free margin.
+     * @return success Boolean indicating if there is sufficient margin to back the debt of the Account.
+     * @return creditor_ The contract address of the Creditor.
+     * @return accountVersion_ The Account version.
+     * @dev An Account is healthy if the collateral value is bigger than or equal to the used margin.
+     */
+    function isAccountHealthy() external view returns (bool success, address creditor_, uint256 accountVersion_) {
```
Comment (Thomas-Smets) [inline src/accounts/AccountV1.sol:391]: Comment Jasper for @acollette, where is this function used?

File: src/accounts/AccountV1.sol [hunk 3]
```diff
@@ -604,8 +614,8 @@ contract AccountV1 is AccountStorageV1, IAccount {
      * to the same asset that will get deposited. If multiple asset IDs of the same contract address
      * are deposited, the assetAddress must be repeated in assetAddresses.
      * Example inputs:
-     * [wETH, DAI, BAYC, Interleave], [0, 0, 15, 2], [10**18, 10**18, 1, 100], [0, 0, 1, 2]
-     * [Interleave, Interleave, BAYC, BAYC, wETH], [3, 5, 16, 17, 0], [123, 456, 1, 1, 10**18], [2, 2, 1, 1, 0]
+     * [wETH, DAI, BAYC, Azuki], [0, 0, 15, 2], [10**18, 10**18, 1, 100], [0, 0, 1, 2]
```
Comment (Thomas-Smets) [inline src/accounts/AccountV1.sol:617]: Azuki is not an ERC1155 right?

File: src/interfaces/IActionBase.sol [hunk 4]
```diff
@@ -16,13 +17,11 @@ struct ActionData {
 }
 
 interface IActionBase {
-    // Struct with information to pass to and from actionHandlers.
-
     /**
-     * @notice Calls a series of addresses with arbitrary calldata.
-     * @param actionData A bytes object containing two actionAssetData structs, an address array and a bytes array.
-     * @return resultData An actionAssetData struct with the balances of this ActionMultiCall address.
+     * @notice Calls a an external target contract arbitrary calldata.
```
Comment (Thomas-Smets) [inline src/interfaces/IActionBase.sol:21]:
```suggestion
     * @notice Calls an external target contract with arbitrary calldata.
```

File: src/accounts/AccountV1.sol [hunk 5]
```diff
@@ -429,6 +392,28 @@ contract AccountV1 is AccountStorageV1, IAccount {
         }
     }
 
+    /**
+     * @notice Checks if the Account is healthy and still has free margin.
```
Comment (j-vp) [inline src/accounts/AccountV1.sol:396]: We more check whether it's unhealthy according to the function name, but acceptable

Comment (j-vp) [reply src/accounts/AccountV1.sol:396]: bool success means then unhealthy? would rephrase comments/functionname/return var

File: src/accounts/AccountV1.sol [hunk 6]
```diff
@@ -508,7 +493,54 @@ contract AccountV1 is AccountStorageV1, IAccount {
...
+
+    /**
+     * @notice Checks that the increase of the open position is allowed.
+     * @param openPosition The new open position.
+     * @return accountVersion The current Account version.
+     * @dev The Account performs the following checks when an open position (liabilities) are increased:
+     *  - The caller is indeed a Creditor for which a margin account is opened.
+     *  - The Account is still healthy after given the new open position.
+     */
+    function increaseOpenPosition(uint256 openPosition) external view onlyCreditor returns (uint256 accountVersion) {
```
Comment (j-vp) [inline src/accounts/AccountV1.sol:507]: if it's a view, do we need the onlyCreditor? Can be good for other integrators/front ends

**Review by j-vp — APPROVED** [review]
gtg after comments are resolve

File: src/accounts/AccountV1.sol (hunk 6 above)
Comment (j-vp) [reply src/accounts/AccountV1.sol:507]: nvm we'll need it to check the creditor address
